Automattic/newspack-plugin
# fix: tooltip position in header and remove duplicated css

### All Submissions:






### Changes proposed in this Pull Request:

Since https://github.com/Automattic/newspack-plugin/pull/1457 there's a tiny bug in the header with the "Return to Dashboard"

I also noticed that we were duplicating the same css, not quite sure why. I've removed it and I couldn't see anything broken.

### How to test the changes in this Pull Request:

1. Check the header
2. Switch to this branch
3. Check the header again
4. Check all the various the various wizards

### Other information:
